bcrs-ui/jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  preset: '@vue/cli-plugin-unit-jest/presets/typescript-and-babel'
+}
```

Comment:
We need to find out why this is different than coops-ui version. Does this one work? Does the coops-ui one need to be updated?
I think that if this requires more than 1 hour of research then we should create a new ticket.

Reply:
As mentioned above, i think this is just a bundle of sorts of the packages mentioned in the name. When you install Jest in the start up of VUE CLI, this is the bundle installed. I believe it saves having to set the `transform` & `fileextensions` options manually.  We can definitely dig into this deeper though, as i don't know what the full suite of options and presets are compared to what was used in the Coops-ui.

Reply:
I had a quick look at this one too, and you're right: it presets some options here. However, coops-ui has some different options.

As above, I think we should "resolve this conversation" but keep an eye on what gets built (ie, npm run build) and how it's loading and running in different browsers, etc. If things seem incorrect then we can revisit this config.